doc/Development_Documentation/02_MVC/00_Controller.md

```diff
@@ -54,8 +54,8 @@ class DefaultController extends FrontendController
      * access to the final response object (as it is automatically created when rendering the view).
      *
      * @Template
```

Comment:
The `@Template` annotation is also not supported anymore, right?

Reply:
Docs updated.